tools/clang/test/CodeGenSPIRV/vk.layout.shader-record-ext.std430.hlsl

```diff
@@ -1,5 +1,8 @@
 // RUN: %dxc -T lib_6_3 -fspv-target-env=vulkan1.2 -fvk-use-gl-layout -fcgl  %s -spirv | FileCheck %s
 
+// CHECK: OpEntryPoint ClosestHitNV %chs1 "chs1" %cbuf %block %P %A
+// CHECK: OpEntryPoint ClosestHitNV %chs2 "chs2" %cbuf %block %P_0 %A_0
```

Comment:
Does this change the location decorations for the input and output variables?

Reply:
I don't think so; I don't think closest hit shaders have any variables in the `Input` or `Output` storage class. (`%cbuf` and `%block` are `ShaderRecordBufferNV`, `%P` is `IncomingRayPayloadNV`, and `%A` is `HitAttributeNV`)